docs/recipes/build/rollup.md

```diff
@@ -0,0 +1,144 @@
+# Rollup-based Bundling
+
+These recipes cover the following `sw-lib` use case:
+- You want to write and maintain your own `sw.js` file that uses precaching via
+`sw-lib.cacheRevisionedAssets()` alongside other service worker logic.
+- You'd like to use [ES2015 import syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import)
+to include `sw-lib` and other modules into your service worker.
+- You are comfortable using [`Rollup`](https://github.com/rollup/rollup) to
+handle transforming the `sw.js` file you maintain into a production-ready,
+self contained service worker file, containing the `sw-lib` library code along
+with your custom logic.
```

Comment:
I'm not going to block because this going to be personal, but I kind of prefer the second half of this intro and think you could go along the lines of:

"If you are using Rollup and ES2015 imports, you can use sw-lib as an import in your service worker and sw-build can output a precache manifest that can be imported as well.

This recipe illustrates how you can use ES2015 imports in your service worker with sw-lib and sw-build."